src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+import { Meta, StoryFn } from "@storybook/react";
+
+import { Tooltip as TooltipComponent } from "./Tooltip";
+import { IconButton } from "../IconButton/IconButton";
+
+import UserIcon from "@vector-im/compound-design-tokens/icons/user.svg";
+
+export default {
+  title: "Tooltip",
+  component: TooltipComponent,
+  argTypes: {},
+  args: {},
+} as Meta<typeof TooltipComponent>;
+
+const Template: StoryFn<typeof TooltipComponent> = () => (
```

Comment:
The story looks a bit wild:

![image](https://github.com/vector-im/compound-web/assets/6216686/1a6f9639-c997-47b9-9f41-38959cedc586)

If possible can you tweak it?

Reply:
We need to discuss stories with the Compound team, and until then I do not think I will spend time changing them as they are more a development sandbox than a real visual documentation.

But that will come.